packages/@aws-cdk/aws-rds/lib/cluster.ts

```diff
@@ -652,16 +667,25 @@ export class DatabaseClusterFromSnapshot extends DatabaseClusterNew {
   public readonly instanceIdentifiers: string[];
   public readonly instanceEndpoints: Endpoint[];
 
+  public readonly secret?: secretsmanager.ISecret;
+
   constructor(scope: Construct, id: string, props: DatabaseClusterFromSnapshotProps) {
     super(scope, id, props);
 
+    const credentials = renderCredentials(this, props.engine, props.credentials);
```

Comment:
I suppose moving credentials to the base props would make sense if this was performed in DatabaseClusterNew as well but I think it makes more sense to leave this here and not move the field.